src/acl/external/time_quota/ext_time_quota_acl.8

```diff
@@ -36,7 +36,7 @@ Default is 300 seconds (5 minutes).
 .
 .if !'po4a'hide' .TP
 .if !'po4a'hide' .B "\-d"
-Enables debug logging to stderr.
+Enables debug logging to stderr, at growing levels of verbosity from 0 to 9.
```

Comment:
Logging to stderr is enabled by default (at level 1). Something like this might work better (there is probably a way to properly reference squid.conf.documented or some such):

```suggestion
Sets verbosity level for debugging section 82 (0-9; defaults to 1; see debug_options).
```

Reply:
I'm not sure I understsand why we should expose the debugging section
number here. If Squid sets it up via environment variables, it's documented
there. But here there is only one section

Reply:
> I'm not sure I understsand why we should expose the debugging section number here.

The suggestion exposes debugging section number because that is one (simple/direct) way to fix a subset of problems identified in my review, including one specific bug identified in this particular change request. 

Needless to say, better solutions (if they exist) are preferred! However, please keep in mind that many of these problems only exist if this PR continues to use `-d`  for controlling a _single_ debugging section (as opposed to all sections).

> here there is only one section

The reader does not know that.

Documentation line this change request is attached to effectively contradicts correct "only one section" assertion because that documentation line talks about logging in general (i.e. _all_ debugging sections). That is not the only problem with that documentation line that should be addressed. The suggested replacement addresses all those problems.

Reply:
@kinkie, I agree with @rousskov on this. the linked libraries used by the helper may produce output for non-82 debug sections. It is useful to be clear that only the section 82 output (of the helper, not of Squid) is affected by `-d level`.